.github/workflows/plugins.yml

```diff
@@ -51,6 +51,11 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set up Go 1.18
```

Comment:
Most of the changes to this file can be reverted after resolving: https://github.com/gojek/turing-experiments/pull/24#discussion_r928360283